src/tcgwars/logic/impl/gen4/LegendsAwakened.groovy

```diff
@@ -332,25 +332,29 @@ public enum LegendsAwakened implements LogicCardInfo {
                 self.cards.moveTo(hand)
                 removePCS(self)
               }
-              // TODO
             }
           }
           move "Pester", {
             text "40+ damage. If the Defending Pokémon is affected by a Special Condition, this attack does 40 damage plus 40 more damage."
             energyCost F, C
-            attackRequirement {}
             onAttack {
-              damage 0
+              damage 40
+              if (!defending.specialConditions) {
```

Comment:
This is checking "if the defending Pokémon has no Special Conditions affecting it," it should probably be just `if (defending.specialConditions)` to check if it is affected by any conditions.

Reply:
What a mistake lmao

Reply:
Fixed